dev/releases/generate_package_release_notes.py

```diff
@@ -90,12 +103,15 @@ def main(old_gap_version, new_gap_version):
     updated = new_json.keys() & old_json.keys()
     updated = [p for p in updated if old_json[p]["Version"] != new_json[p]["Version"]]
     if len(updated) > 0:
-        print(f"""
+        print(
+            f"""
 #### Updated packages redistributed with GAP
 
-The GAP {new_gap_version} distribution contains {len(new_json)} packages, of which {len(updated)} have been
-updated since GAP {old_gap_version}. The full list of updated packages is given below:
-""".lstrip())
+The GAP {new_gap_version} distribution contains {len(new_json)} packages, of
```

Comment:
This shouldn't be rewrapped like that, as it changes the line breaks in the final file: in the generated file, `{len(new_json)}` is replaced by 3 digits, and `{len(updated)}` by 1 to 3, etc..

Reply:
I've undone that too